src/components/SelectedMovie/SelectedMovie.css

```diff
@@ -1,6 +1,72 @@
+/* .backdrop {
+    width: auto;
+    height: 60vh;
+    overflow: hidden;
+    object-fit: cover;
+} */
+
 .backdrop {
+    border-radius: 1em;
+    box-shadow: 7px 2px 20px 0px;
+    padding: 10px 10px 10px 10px; 
     width: auto;
     height: 60vh;
     overflow: hidden;
     object-fit: cover;
+  }
+
+  .backdrop-container {
+      display: flex;
+      justify-content: flex-end;
+  }
+
+  .poster-container {
+    margin-top: 1em;
```

Comment:
This is a great way to solve the problem we were having with the poster/title/overview formatting. I think we can also add some margins on the sides to keep the information/poster more centered/easier to read.